### PR TITLE
Consent attribute added for Physical card , share credential and Update UIN API (23507)

### PR DESCRIPTION
**Pull request by Neha2365**

### CODE REVIEW

File: resident/resident-service/src/main/java/io/mosip/resident/service/impl/OrderCardServiceImpl.java [hunk 1]
```diff
@@ -63,35 +63,58 @@ public class OrderCardServiceImpl implements OrderCardService {
...
 			throws ResidentServiceCheckedException, ApisResourceAccessException {
 		logger.debug("OrderCardServiceImpl::sendPhysicalCard()::entry");
 		ResidentCredentialResponseDto residentCredentialResponseDto = new ResidentCredentialResponseDto();
 
-		 ResidentTransactionEntity residentTransactionEntity = createResidentTransactionEntity(requestDto);
-		
-		if (isPaymentEnabled) {
-			checkOrderStatus(requestDto.getTransactionID(), requestDto.getIndividualId(), residentTransactionEntity);
+		ResidentTransactionEntity residentTransactionEntity = createResidentTransactionEntity(requestDto);
+		if (requestDto.getConsent().equalsIgnoreCase(ConsentStatusType.DENIED.name())
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/service/impl/OrderCardServiceImpl.java]: Add null check here.

File: resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java [hunk 2]
```diff
@@ -908,8 +917,23 @@ public ResidentUpdateResponseDTO reqUinUpdate(ResidentUpdateRequestDto dto) thro
 					dto.getTransactionID(), "Request for UIN update"));
 			throw new ResidentServiceException(ResidentErrorCode.BASE_EXCEPTION.getErrorCode(),
 					ResidentErrorCode.BASE_EXCEPTION.getErrorMessage(), e);
-		} finally {
-			if(Utilitiy.isSecureSession()) {
+
+		}
+
+		try {
+			if (dto.getConsent().equalsIgnoreCase("Denied") || dto.getConsent().isEmpty()
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java]: Add null check here for dto.getConsent()

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java (hunk 2 above)
Comment (Neha2365) [reply resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java]: Already checked

File: resident/resident-service/src/main/java/io/mosip/resident/service/impl/OrderCardServiceImpl.java (hunk 1 above)
Comment (Neha2365) [reply resident/resident-service/src/main/java/io/mosip/resident/service/impl/OrderCardServiceImpl.java]: done

**Review by loganathan-sekaran — APPROVED** [review]

**Pull request merged**